external-import/misp/src/config.yml.sample

```
@@ -18,7 +18,9 @@ misp:
   key: 'ChangeMe' # Required
   ssl_verify: true # Required
   client_cert: '' # Optional, Path to the client certificate eg. /app/cert/clientcert.pem
-  datetime_attribute: 'timestamp' # Required, filter to be used in query for new MISP events
+  MISP_DATE_FILTER_FIELD=from_date
```

```suggestion
  date_filter_field=from_date
```

To align with other variables from config.yml.sample